pkgs/games/ldmud/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib, fetchFromGitHub, stdenv, autoreconfHook, pkg-config, bison, libiconv
+, pcre, libgcrypt, ipv6Support ? false, mccpSupport ? false, zlib
+, mysqlSupport ? false, libmysqlclient, postgresSupport ? false, postgresql
+, sqliteSupport ? false, sqlite, json_c, libxml2, tlsSupport ? false, openssl
+, pythonSupport ? false, python310 }:
```

Comment:
Separate the `*Support` args? See [an example](https://github.com/ckiee/nixpkgs/blob/8709490e60f42c4fcd402eb96cdfaa93f86669ae/pkgs/development/libraries/ffmpeg/generic.nix#L1-L26).

Reply:
Looks nicer, thanks! Fixed in e651f31